docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -98,6 +98,86 @@ may prevent the limit from being reached. Running queries on the ``information_s
 table without a schema and table name filter may lead to hitting the limit, as this requires
 fetching the sheet data for every table, unless it is already cached.
 
+Type mapping
+------------
+
+Because Trino and Google Sheets each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading data.
+
+Google Sheets type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Google Sheets types to the corresponding Trino types
+following this table:
+
+.. list-table:: Google Sheets type to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Google Sheets type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``ARRAY``
+    - ``ARRAY``
+    -
+  * - ``STRING``
+    - ``VARCHAR``
+    -
+  * - ``TEXT``
+    - ``CHAR``
+    -
+  * - ``DATE``
+    - ``DECIMAL (p,s)``
+    -
+  * - ``JSON``
+    - ``JSON``
+    -
+
+No other types are supported.
+
+Trino type to Google Sheets type mapping
```

Comment:
The connector is read-only for now. Let's remove this mapping table. Write operations will be supported in https://github.com/trinodb/trino/pull/5011